### DISCUSSION TITLE
Frigate+ - how coupled are models to specific cameras?

### DISCUSSION THREAD

**Discussion by TwoBigPaws** [blakeblackshear/frigate#15528] [Ask A Question]
I am in my early days of HA and Frigate+, so my camera locations are likely not in their final positions (which may take many months to do....). AND I have much to learn about both platforms.

I am trying to understand if I am wasting my time labelling images on cameras I am likely to re-locate? The new camera locations may be similar, but could be quite different as well...

Or if I introduce a new camera to overlook a similar location (but from a vastly different angle), will the prior train of a model via images on another camera significantly benefit the new camera?

Furthermore (and likely answered by the prior questions' answers...), if I label an object in one camera, will it significantly help my other cameras detect the same object?

Many thanks!
-AB

**Comment by NickM-27** [comment]
I've moved cameras at least a few feet if not more multiple times and never had any issues
